tools/dependencies.props

```diff
@@ -10,6 +10,6 @@
     <LibuvVersion>1.9.1</LibuvVersion>
     <TestSdkVersion>15.0.0</TestSdkVersion>
     <XunitVersion>2.2.0</XunitVersion>
-    <BenchmarkDotNetVersion>0.11.1</BenchmarkDotNetVersion>
+    <BenchmarkDotNetVersion>0.11.1.776</BenchmarkDotNetVersion>
```

Comment:
Why this particular build

Reply:
It's the latest BDN build available (which contains support for netcoreapp3.0).